Jenkinsfile

```diff
@@ -140,11 +142,15 @@ spec:
     imagePullPolicy: Always
     env:
       - name: JAVA_TOOL_OPTIONS
-        value: -Xmx6G
+        value: -Xmx1G
     resources:
       limits:
-        memory: "10Gi"
-        cpu: "2.0"
+        memory: "6Gi"
+        cpu: "1.0"
```

Comment:
We are reducing the max memory size from 10gb to 6gb per container/vm used for TCK testing.  We are also going from 2 CPU cores per TCK test run, to 1 CPU core (we shouldn't need a second CPU much, imo).